sw_utils/ipfs.py

```diff
@@ -37,13 +28,16 @@ def __init__(
         self.username = username
         self.password = password
 
-    async def upload(self, data: bytes) -> str:
+    async def upload_bytes(self, data: bytes) -> str:
+        return await self.upload_json(_to_json(data))
```

Comment:
We can use `add_bytes` method

Reply:
Don't quite get it. Do you suggest renaming upload_bytes to add_bytes?